src/main/java/net/fabricmc/installer/server/ServerInstaller.java

```diff
@@ -18,16 +18,7 @@
 
 import net.fabricmc.installer.util.*;
 
-import java.io.BufferedReader;
-import java.io.BufferedWriter;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.io.OutputStream;
-import java.io.OutputStreamWriter;
+import java.io.*;
```

Comment:
this all import is my only nitpick

Reply:
I went through *all* files and removed star and unused imports.